include/sys/dsl_pool.h

```diff
@@ -56,6 +56,8 @@ struct dsl_crypto_params;
 
 extern unsigned long zfs_dirty_data_max;
 extern unsigned long zfs_dirty_data_max_max;
+extern unsigned long dirty_data_max;
+extern unsigned long dirty_data_max_max;
```

Comment:
Rather than making these global variables public, let's keep them internal to `dsl_pool.c` and add some accessor functions.